code/modules/mob/living/carbon/human/human.dm

```diff
@@ -97,90 +97,75 @@
 			stat("Gnawing hunger", "[carrion_hunger]/[round(maw_efficiency/10)]")
 
 		var/obj/item/implant/core_implant/cruciform/C = get_core_implant(/obj/item/implant/core_implant/cruciform)
-		if (C)
+		if(C)
 			stat("Cruciform", "[C.power]/[C.max_power]")
 
 /mob/living/carbon/human/ex_act(severity, epicenter)
 	if(!blinded)
-		if (HUDtech.Find("flash"))
+		if(HUDtech.Find("flash"))
 			flick("flash", HUDtech["flash"])
 
 	var/b_loss = 0
 	var/bomb_defense = getarmor(null, ARMOR_BOMB) + mob_bomb_defense
 	var/target_turf // null means epicenter is same tile
-	if (epicenter != get_turf(src))
+	if(epicenter != get_turf(src))
 		target_turf = get_turf_away_from_target_simple(src, epicenter, 8)
 	var/throw_distance = 8 - 2*severity
 	var/not_slick = TRUE
-	if (target_turf) // this means explosions on the same tile will not fling you
+	if(target_turf) // this means explosions on the same tile will not fling you
 		throw_at(target_turf, throw_distance, 5)
 		not_slick = FALSE // only explosions that fling you can be survived with slickness
-	if (slickness < (9-(2*severity)) * 10)
+	if(slickness < (9-(2*severity)) * 10)
 		Weaken(severity) // If they don't get knocked out , weaken them for a bit.
 		not_slick = TRUE // if you don't have enough slickness, you can't safely ride the boom
 	else
 		slickness -= (9-(2*severity)) * 10 // awesome feats aren't something you can do constantly.
```

Comment:
draining is done here, not where the damage is decided, please either refactor or undo those changes to avoid there being slickness drained for nothing.

Reply:
If player even going to survive severity-two explosion, I bet that being blown up is not good for one's style.

Reply:
120 damage concentrated on chest and groin is survivable.

Reply:
But does it make you stylish?

Reply:
Surviving it is pretty badass, I guess